jira/client.py

```diff
@@ -4295,15 +4508,15 @@ def add_issues_to_sprint(self, sprint_id, issue_keys):
                 % self._options["agile_rest_path"]
             )
 
-    def add_issues_to_epic(self, epic_id, issue_keys, ignore_epics=True):
+    def add_issues_to_epic(
+        self, epic_id: int, issue_keys: str, ignore_epics: bool = True
+    ) -> Response:
         """Add the issues in ``issue_keys`` to the ``epic_id``.
 
-        :param epic_id: The ID for the epic where issues should be added.
-        :type epic_id: int
-        :param issue_keys: The issues to add to the epic
-        :type issue_keys: str
-        :param ignore_epics: ignore any issues listed in ``issue_keys`` that are epics. (Default: True)
-        :type ignore_epics: bool
+        Args:
+            epic_id (int): The ID for the epic where issues should be added.
```

Comment:
#968 mentions a mistake here maybe take it along

Reply:
close the issue if you take it along

Reply:
I've added it as a linked issue, I'll close after this PR is merged